editor.planx.uk/src/ui/shared/Autocomplete/components/RenderOptionCheckbox.tsx

```diff
@@ -21,6 +21,7 @@ export const RenderOptionCheckbox = ({
     <ListItem key={key} {...restListProps}>
       <CustomCheckbox
         aria-hidden="true"
+        aria-selected={state.selected ? "true" : "false"}
```

Comment:
```suggestion
        aria-selected={state.selected}
```

nit: A little cleaner maybe?

I always find that these `Booleanish` DOM types a little slower to parse as seeing a string "true" or "false" always makes you second guess their types!